docs/src/app/navigationConstants.js

```diff
@@ -392,6 +392,17 @@ export const navigationItems = [
                 path: 'inputs',
             },
             {
+                directory: 'lab',
+                label: 'Lab',
```

Comment:
Just curious: Why "Lab" as the category for the Timeline?

Reply:
That is where MUI put's this component. It's there beta bucket to say, "hey, these components may have issues. Use at your own risk."